platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/migration/ExportMigrationManagerImpl.java

```diff
@@ -156,20 +172,32 @@ public void doExport(String productBranding, String productVersion) {
     LOGGER.debug("Exported metadata: {}", metadata);
   }
 
+  @SuppressWarnings({ //
+    "squid:S2093", /* try-with-resource would require that the input stream is initialized too early */
+    "squid:S2095" /* stream is closed in finally clause */
```

Comment:
❗️ This suppression is no longer needed/valid.

Reply:
Removed